src/main/java/core/basesyntax/service/FruitServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service;
+
+import static java.util.stream.Collectors.toList;
+
+import core.basesyntax.model.Fruit;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitServiceImpl implements FruitService {
+
+    private final List<Fruit> storage = new ArrayList<>();
```

Comment:
according to OC principle storage should be separate class and inject it to the constructor of this class for the case if we will want to implement another type of storage

Reply:
agree